docs/references/phpdoc/tags/param.rst

```diff
@@ -1,57 +1,53 @@
 @param
 ======
 
-The @param tag is used to document a single argument of a function or method.
+The ``@param`` tag is used to document a single `argument of a function or method`_.
 
 Syntax
 ------
 
+.. code-block::
+
     @param [Type] [name] [<description>]
 
 Description
 -----------
 
-With the @param tag it is possible to document the type and function of a
-single argument of a function or method. When provided it MUST contain a
-Type to indicate what is expected; the description on the other hand is
-OPTIONAL yet RECOMMENDED in case of complicated structures, such as associative
-arrays.
+With the ``@param`` tag it is possible to document the :doc:`Type <../types>`
+and function of a single `argument of a function or method`_. When provided it
+MUST contain a :doc:`Type <../types>` to indicate what is expected.
+The description is OPTIONAL yet RECOMMENDED, for instance, in case of
+complicated structures, such as associative arrays.
 
-The @param tag MAY have a multi-line description and does not need explicit
+The ``@param`` tag MAY have a multi-line description and does not need explicit
 delimiting.
 
 It is RECOMMENDED when documenting to use this tag with every function and
-method. Exceptions to this recommendation are:
+method.
 
 This tag MUST NOT occur more than once per argument in a PHPDoc and is
-limited to Structural Elements of type method or function.
+limited to *Structural Elements* of type method or function.
 
 Effects in phpDocumentor
 ------------------------
 
-Structural Elements of type method or function, that are tagged with the
-@param tag, will have additional information in the section regarding Arguments.
+*Structural Elements* of type method or function, that are tagged with the
+``@param`` tag, will have additional information in the section regarding Parameters.
 
-If the return Type is a class that is documented by phpDocumentor, then
-a link to that class' documentation is provided.
+If the parameter *Type* is a class that is documented by phpDocumentor,
+then a link to that class' documentation is provided.
 
 .. note::
 
-   phpDocumentor supports @param tags which omit the name, this is
+   phpDocumentor supports ``@param`` tags which omit the name, this is
```

Comment:
Are we able to do this only in cases where there are param tags given for every arg in the code signature?  If so, we should highlight that requirement... otherwise we're saying here that my earlier `foo()` isn't true, and that we _can_ resolve this arg the one param is tied to.

So:
`phpDocumentor supports @param tags which omit the name, but only if every argument is given a param tag.`